src/main/java/com/google/cloud/teleport/templates/common/SplunkConverters.java

```diff
@@ -116,6 +116,13 @@ public static FailsafeStringToSplunkEvent failsafeStringToSplunkEvent(
     ValueProvider<Boolean> getIncludePubsubMessage();
   
     void setIncludePubsubMessage(ValueProvider<Boolean> includePubsubMessage);
+
+    @Description(
+        "KMS Encryption Key should be in the format "
+            + "projects/{gcp_project}/locations/{key_region}/keyRings/{key_ring}/cryptoKeys/{kms_key_name}")
+    ValueProvider<String> getKMSEncryptionKey();
```

Comment:
can this parameter be named `--tokenKMSEncryptionKey`?
reasoning: to disambiguate that this is not used by to decrypt pubsub data which is handled fluently by beam https://cloud.google.com/dataflow/docs/guides/customer-managed-encryption-keys#pubsub

Reply:
Done